mwdb/core/mail.py

```diff
@@ -57,8 +57,18 @@ def send_email_notification(kind, subject, recipient_email, **params):
     else:
         smtp_host = mail_smtp
         smtp_port = 25
+    
+    username = app_config.mwdb.mail_username
+    password = app_config.mwdb.mail_password
+    tls = app_config.mwdb.mail_tls
+
     try:
         with smtplib.SMTP(smtp_host, int(smtp_port), timeout=3) as s:
-            s.send_message(message)
+            if tls:
+                s.starttls()
+ 
+            if username and password:
+                s.login(username,password)
+                s.send_message(message)
```

Comment:
What if `username` and `password` are not provided? `s.send_message(message)` will not execute, because it's inside the condition block

Reply:
You are right. unindenting works though.
Approved @ITAYC0HEN 's fix.

Reply:
Already fixed by previous suggestion